recipes/pdfcombine/meta.yaml

```diff
@@ -0,0 +1,42 @@
+{% set name = "pdfcombine" %}
+{% set version = "0.3.2" %}
+{% set sha256 = "65d57f7024ce51c1f1c9123a1d71085288d5cec145d830b9a145d5c4e335f9ee" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://github.com/tdegeus/pdfcombine/archive/v{{ version }}.tar.gz
+  sha256: {{ sha256 }}
+
+build:
```

Comment:
Since the steps are same for Windows and Unix, you can remove `build.sh` and `bld.bat` completely, and use:
```
build:
  number: 0
  noarch: generic
  script:
    - mkdir ${PREFIX}/bin/
    - cp pdfcombine ${PREFIX}/bin/
    - chmod +x ${PREFIX}/bin/pdfcombine
```
here. Can you please go ahead and try that?

Reply:
@npavlovikj For my understanding: how does it know to do the right thing in Windows? In particular the last line (as Windows to my knowledge does not really have permissions).

Reply:
@npavlovikj In fact is does not know what to do in Windows (see CI)